public_officials/views.py

```diff
@@ -15,7 +16,8 @@ def home_page(request):
 
 def legislator_detail(request, legislator_id):
     legislator = Legislator.objects.get(pk=legislator_id)
-    return render(request, 'public-officials/detail.html', {'legislator': legislator})
+    followers = legislator.users.all()
```

Comment:
we could refactor this out later to check if the user is in the follower list and then pass in a boolean instead of an entire hash